dashboard/src/components/Catalog/PackageCatalogItem.tsx

```diff
@@ -15,17 +15,24 @@ export default function PackageCatalogItem(props: IPackageCatalogItem) {
     availablePackageSummary.availablePackageRef!,
   );
 
-  // Historically, this tag is used to show the repository a given package is from,
-  // but each plugin as its own way to describe the repository right now.
-  let repositoryName;
+  let pkgRepository;
+  let pkgPlugin;
+
   switch (availablePackageSummary.availablePackageRef?.plugin?.name) {
     case PluginNames.PACKAGES_HELM:
-      repositoryName = availablePackageSummary.availablePackageRef?.identifier.split("/")[0];
+      pkgRepository = availablePackageSummary.availablePackageRef?.identifier.split("/")[0];
+      pkgPlugin = "helm";
+      break;
+    case PluginNames.PACKAGES_FLUX:
+      // TODO: get repo from flux
+      pkgPlugin = "flux";
+      break;
+    case PluginNames.PACKAGES_KAPP:
+      pkgPlugin = "carvel";
```

Comment:
Perhaps another update to the plugin API, which currently has the plugin details just with the machine-readable name (fluxv2.packages or kapp-controller.packages)... we could think about adding a displayname to avoid the dashboard needing to know about packaging plugins here.

Reply:
Yep, good idea indeed. I've added a card not to forget. Thanks!